OPHD/UI/StructureInspector.cpp

```diff
@@ -161,7 +162,34 @@ void StructureInspector::update()
 	}
 
 	drawPopulationRequirements();
+	drawStructureTypeSpecific();
 
-	position = mRect.startPoint() + NAS2D::Vector{5, static_cast<int>(mRect.height()) - FONT->height() - 5};
+	position = mRect.startPoint() + NAS2D::Vector{ 5, static_cast<int>(mRect.height()) - FONT->height() - 5 };
```

Comment:
I think this project tends to use tight braces pretty consistently when initializing structs. I haven't really verified though, and there hasn't been a formal discussion on the style.

Reply:
Personally I prefer the spacing in the updated line, for whatever reason it's easier to read. Might be worth making that consistent.

Reply:
I did some quick grepping on this. Searching for tight braces reveals a whole lot of data structures and classes being initialized. Grepping for loose braces reveals a whole lot of inline function definitions. There were some exceptions in each case though.


For code with tight braces, I see quite a few examples in initializer lists, such as:
```cpp
PlanetSelectState::PlanetSelectState() :
	mBg{"sys/bg1.png"},
	mStarFlare{"sys/flare_1.png"},
	mDetailFlare{"sys/flare_2.png"},
	mDetailFlare2{"sys/flare_3.png"},
	mCloud1{"sys/cloud_1.png"},
	mCloud2{"sys/cloud_2.png"},
	mBgMusic{"music/menu.ogg"},
	mSelect{"sfx/click.ogg"},
	mHover{"sfx/menu4.ogg"},
	mQuit{"Main Menu"},
	mReturnState{this}
{}
```

I think previously the code used `()` in a lot of cases, but was updated to use `{}` instead.

For reference, there's hardly any code that includes a space after an opening parenthesis. Aside from the micropather code, there are only two instances of that, one of which is a `#pragma message( ...`.


I think the particular case quoted is a bit uglier because of the calculation done inside the `{}`. Most cases are much simpler, such as:
```cpp
mPlanetDescription.size({550, 200});
```

Or:
```cpp
renderer.drawImageRotated(mCloud1, {-256, -256}, rotation, NAS2D::Color{100, 255, 0, 135});
```

Not sure if we'd want to add spaces there. I suspect extra spacing around simple cases could make them harder to read.


For an example of loose braces, they're mostly used in header files for inline function defintions:
```cpp
PlanetType type() const { return mType; }
```

Reply:
I'm not sure it is worth regulating space or no space for this situation.

It was sloppy of me to add commit noise and not catch it. I suspect Visual Studio's default formatting made the line change when I was refactoring.